modules/admission-plug-ins-default.adoc

```diff
@@ -6,4 +6,73 @@
 = Default admission plug-ins
 
 //Future xref - A set of default admission plug-ins is enabled in {product-title} {product-version}. These default plug-ins contribute to fundamental control plane functionality, such as ingress policy, xref:../nodes/clusters/nodes-cluster-overcommit.adoc#nodes-cluster-resource-override_nodes-cluster-overcommit[cluster resource limit override] and quota policy.
-A set of default admission plug-ins is enabled in {product-title} {product-version}. These default plug-ins contribute to fundamental control plane functionality, such as ingress policy, cluster resource limit override and quota policy.
+Default validating and admission plug-ins are enabled in {product-title} {product-version}. These default plug-ins contribute to fundamental control plane functionality, such as ingress policy, cluster resource limit override and quota policy. The following lists contain the default admission plug-ins:
```

Comment:
Something about _The following lists contain _ doesn't sound right.  _The following are the default admission lug-ins:_??

Reply:
Every instance of "following" in the "Lead-in wording" section of the ISG is followed by a noun.